### ISSUE TITLE
Dependency versions

### ISSUE THREAD

**Issue by supersciencegrl** [thomas0809/MolScribe#29]
Hi team, 

Thanks for making this pretty awesome library! 

An issue I'm having installing it on a new PC using pip - the version <2.0 of PyTorch that's required only works with old versions of Python. It seems that you need python < 3.11 for torch < 2.0. 

Also noticed that the torch version isn't specified in requirements.txt - but it is specified when you try to install with pip. 

Worth adding python < 3.11 and torch < 2.0 to requirements.txt, if these are the right requirements? 

Thanks very much! 
-Nessa

**Comment by LuHC409** [comment]
Yeah, I ran into the same thing! The issue is that PyTorch < 2.0 only supports up to Python 3.10, so you'll need to make sure your environment uses something like Python 3.9. The easiest way I found is to use conda to create a virtual environment, like this:
conda create -n myenv python=3.9
conda activate myenv

Then install PyTorch separately, depending on your GPU and CUDA version. Since the right PyTorch version varies depending on your hardware, it’s probably why it’s not fixed in requirements.txt.

Hope that helps! :)

**Comment by supersciencegrl** [comment]
Thank you LuHC409, I will try that! I didn't realize this before. I will give it another go :)
Still could be worth mentioning in installation documentation for people like me who may not come in knowing these things :))

**Comment by MauricioCafiero** [comment]
I have not been able to install as pip can't find torch below 2.3 I think. Anyone running on Colab willing to share how they installed?

**Comment by flange-ipb** [comment]
The dependency updates presented in #37 work for me (using Python 3.11 and _uv_ as package manager).

**Comment by thk9178** [comment]
This issue is actually already resolved on the `main` branch. The `torch<2.0` constraint was removed in commit `11184a0` (2023-08-09), just 4 months after the 1.1.1 PyPI release that added it.

As a workaround, installing from Git source works fine with Python 3.11 + torch 2.x:
```
pip install git+https://github.com/thomas0809/MolScribe.git
```

It would be great if a new version could be published to PyPI to reflect this change. The current 1.1.1 on PyPI is the only blocker preventing straightforward `pip install molscribe` on modern Python environments.

**Comment by alexey-krasnov** [comment]
One could solve the issue by using Python 3.11 or lower with the dependency updates presented in [#37](https://github.com/thomas0809/MolScribe/pull/37). 

The update of Molscribe to Python>3.11 needs resolving in **OpenNMT-py** and **albumentations** dependencies.

**Comment by flange-ipb** [comment]
> The update of Molscribe to Python>3.11 needs resolving in **OpenNMT-py** and **albumentations** dependencies.

@alexey-krasnov 
I think it is worth looking at @lucas-morin's [MolScribe fork](https://github.com/thomas0809/MolScribe/compare/main...lucas-morin:MolScribe:main). This one is used by [MarkushGrapher](https://github.com/DS4SD/MarkushGrapher).
